CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 - Reactive components have been removed completely. [#1195](https://github.com/spatialos/gdk-for-unity/pull/1195)
   - If you are using reactive components, please see our documentation on the [ECS workflow](https://docs.improbable.io/unity/alpha/workflows/overview#ecs-centric-workflow).
 
+### Changed
+
+- Generated Worker ID's for local development are now smaller and easier to read for debugging. [#1197](https://github.com/spatialos/gdk-for-unity/pull/1197)
```

Comment:
Is this just for local development?

Reply:
Yes, for all other deployments the runtime generates your workerid